### PR TITLE
add console log export details for go

### PR DESCRIPTION
**Pull request by codeboten**
This follows the examples for trace/metrics and marks log export support as experimental.

---

**Preview**: https://deploy-preview-4357--opentelemetry.netlify.app/docs/languages/go/exporters/#console

### CODE REVIEW

File: content/en/docs/languages/go/exporters.md [hunk 1]
```diff
@@ -49,6 +49,24 @@ func newExporter() (metric.Exporter, error) {
 }
 ```
 
+### Console logs (Experimental)
```
Comment (chalin) [inline content/en/docs/languages/go/exporters.md:52]:
```suggestion
### Console logs (Experimental) {#console-logs}
```

**Comment by codeboten** [comment]
Finally passes all the CI checks :D Thanks for the review @chalin

**Review by chalin — APPROVED** [review]

**Comment by cartermp** [comment]
@open-telemetry/go-approvers PTAL!

**Review by MrAlias — APPROVED** [review]

**Pull request merged**